servicetalk-grpc-internal/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2019, 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+  api project(":servicetalk-concurrent-api")
+  api project(":servicetalk-http-api")
+  api project(":servicetalk-router-api")
+  api ("com.google.api.grpc:proto-google-common-protos:$protoGoogleCommonProtosVersion") {
```

Comment:
Is this needed, and if so, api?

Reply:
consider reviewing the dependencies here, I think you only need
api `concurrent-api`
api `http-api`
implementation `buffer-api`
implementation `utils-internal`

implementation "org.slf4j:slf4j-api:$slf4jVersion"
implementation "com.google.code.findbugs:jsr305:$jsr305Version"

+ test deps